### PR TITLE
Add missing serverspec:windows test cases

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by cosmo0920** [comment]
Hmm..., serverspec on Windows still failing....: https://github.com/fluent-plugins-nursery/td-agent-builder/runs/1019767415#step:5:421

File: serverspec/windows/td-agent.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      end
+    end
+  end
+end
+
+config_path = File.join(File.dirname(File.dirname(File.dirname(__FILE__))),
+                        "td-agent/config.rb")
+File.open(config_path) do |file|
+  eval(file.read)
+end
```
Comment (cosmo0920) [inline serverspec/windows/td-agent.rb:23]: `eval` is dangerous.

```ruby
require config_path
```

is enough to load config.rb.

**Comment by kenhys** [comment]
https://github.com/fluent-plugins-nursery/td-agent-builder/pull/180/checks?check_run_id=1020161022#step:5:791
td and webhdfs is not detected correctly.

**Comment by cosmo0920** [comment]
We got green for Serverspec on Windows! 💚

**Review by cosmo0920 — APPROVED** [review]
LGTM!

**Pull request merged**